### PR TITLE
Add tiddlywiki.svg

### PR DESCRIPTION
**Pull request by oeyoews**
**Issue:** closes #

**Similarweb rank:** [7.2k Github Stars](https://github.com/Jermolene/TiddlyWiki5)
  
image

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

add tiddlywiki logo

### CODE REVIEW

**Comment by Jermolene** [comment]
Hi @oeyoews thank you for submitting the TiddlyWiki icon.

Unfortunately, it has not come out correctly – the image proportions have been changed so that it looks squashed vertically.

I believe that the problem is that the viewbox on the submitted SVG has been made into a square, but the underlying icon is not square (the proportions should be 2115x2481). I think it can be fixed by adding some vertical padding within the viewbox. Would you like me to prepare a fixed version?

**Comment by Jermolene** [comment]
Thanks @oeyoews here's a new version of the icon that is properly centered in a square viewbox. Would you kindly be able to make the changes required by this repo, and update this PR?

TiddlyWiki Hex Icon

**Comment by oeyoews** [comment]
image

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Tiddlywiki</title><path d="m12 0 12 6v12l-12 6-12-6V6Zm4.66 17.88.19-.02c.11-.05.26-.09.33-.2.27-.32-.42-.46-.66-.82-.25-.36-1.26-2.36-.5-2.92.77-.56 1.6-.8 1.92-1.31.12-.24.27-.47.32-.74.17.13.33.25.59.26.64.04 1.22-.66 1.53-1.3.15-.42.28-.85.26-1.25-.09-.75 0-1.57.7-2.08-.7.3-1.36.69-2.16.8a4.12 3.5 0 0 0-1.36-.21c-.2-.26-.51-.72-.91-.78-.69.2-.71 1.1-1 2.3a7.7 7.7 0 0 0-1.34-.18c-.64-.03-1.67.14-2.4.12C11.46 9.53 10 9 8.82 9.1c-.47.05-.96.12-1.4.3-.28.1-.67.42-.87.46-1.97.47-5.2.46-4.55-2.1.06-.2.07-.37-.03-.41-.12-.06-.2.06-.27.18-1.16 2.14.89 3.51 3.23 3.52.73-.05.38 0 1.06-.12l-.02.08c-.06.7.33 1.36 1.02 1.73-.32.76-.94 1.38-1.38 2.09.24.87.83 1.65 1.65 2.22.38.15 1.21.15 1.41-.09.06-.1 0-.35-.06-.47.32.3.77.73 1.18.94.25.1.52.17.8.18l.35-.02c.2-.03.41-.05.53-.2.14-.2 0-.75-.55-.8-.57-.03-.94-.2-1.46-.87-.42-.63.1-1.5.27-2.14.9.12 1.8.22 2.7.22a8.35 7.1 0 0 0 1.18-.1c.12.28.47 1.03.47 1.25 0 .6-.24 1.2-.13 1.8.06.18 0 .5.4.52l.47.02h.3c.4.57.75.57 1.54.6zm-8.82-1.66-.09-.06c-.2-.2-.76-.54-.67-.86.1-.24.26-.46.38-.68v.02c-.02.28-.07.56.05.84.31.22.76.6 1.06.83-.16-.04-.47-.11-.62-.06z"/></svg>
```

It seems to be a bit flat this time

**Comment by Jermolene** [comment]
> It seems to be a bit flat this time

Yes indeed, that is still not correct. What are the steps that were taken to convert the SVG that I [provided](https://github.com/simple-icons/simple-icons/pull/8326#issuecomment-1445182286) into this new version?

**Comment by oeyoews** [comment]
First of all, I need to adjust the size of the picture to 24, which is easy to do with inkscape, but I can't understand the style attribute, so I continue to use svgo for processing, only retaining a path, but the picture is no longer square at this time, is it possible Is the problem caused by this svg size is 283.464x283.466, or svgo @Jermolene

File: icons/tiddlywiki.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Tiddlywiki</title><path d="m12 0 12 6v12l-12 6-12-6V6zm4.64 17.89.19-.02c.13-.04.25-.08.33-.19.27-.33-.42-.46-.66-.83-.25-.36-1.26-2.36-.5-2.92.77-.56 1.6-.8 1.92-1.32.14-.23.28-.47.31-.74.19.13.34.25.59.26.66.04 1.23-.65 1.53-1.31.16-.4.29-.83.26-1.23-.08-.76.01-1.58.72-2.08-.7.3-1.37.69-2.16.79a4.72 4.72 0 0 0-1.34-.2c-.23-.26-.54-.73-.94-.79-.68.21-.7 1.1-1 2.33a7.14 7.14 0 0 0-1.34-.2c-.64-.04-1.67.13-2.4.12-.72-.02-2.15-.57-3.33-.45-.48.05-.97.12-1.4.3-.28.1-.67.41-.87.46-1.98.47-5.21.45-4.55-2.1.06-.2.07-.37-.03-.42-.13-.06-.2.06-.28.19-1.16 2.13.9 3.51 3.23 3.52.73-.05.38 0 1.06-.12l-.02.08c-.04.71.33 1.34 1.02 1.72-.32.77-.95 1.39-1.38 2.1.16.88 1.21 2.06 1.64 2.23.39.14 1.22.14 1.41-.1.07-.1.02-.35-.05-.47.32.3.77.74 1.19.94.25.1.51.17.8.18l.36-.01c.18-.04.39-.06.51-.2.14-.2.01-.76-.55-.8-.57-.03-.94-.22-1.46-.88-.43-.63.1-1.51.26-2.14.9.12 1.8.22 2.71.22.4-.02.8-.03 1.18-.1.12.28.47 1.03.47 1.24.01.6-.25 1.21-.14 1.81.07.18 0 .52.41.53.25 0 .1 0 .46.02l.31-.01c.4.57.75.57 1.53.59zm-8.82-1.66-.07-.06c-.22-.2-.77-.54-.67-.86.09-.24.25-.46.38-.68v.02c-.03.28-.08.56.03.83.32.23.78.6 1.08.84-.17-.03-.47-.12-.63-.07Z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/tiddlywiki.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Tiddlywiki</title><path fill-rule="evenodd" d="m12 0 10.23 6v12L12 24 1.77 18V6L12 0zm3.961 17.889.154-.02c.113-.043.22-.081.288-.19.227-.329-.357-.462-.566-.827-.209-.364-1.071-2.364-.418-2.924s1.359-.79 1.629-1.315c.117-.236.238-.475.269-.742.159.132.283.255.497.262.567.036 1.054-.658 1.307-1.315.135-.404.244-.832.218-1.226-.069-.76.013-1.582.62-2.087-.599.302-1.167.69-1.845.789-.374-.114-.75-.216-1.147-.2-.194-.253-.456-.727-.797-.782-.58.208-.597 1.105-.842 2.321a5.351 5.351 0 0 0-1.154-.193c-.54-.035-1.42.134-2.038.116-.619-.018-1.836-.562-2.849-.445-.407.05-.817.12-1.195.291-.231.105-.565.421-.733.468-1.69.473-4.442.453-3.879-2.102.044-.196.056-.373-.03-.417-.11-.055-.17.06-.234.187-.985 2.138.764 3.514 2.752 3.52.625-.048.324-.007.904-.118l-.015.082a1.87 1.87 0 0 0 .865 1.718c-.27.771-.805 1.389-1.173 2.097.138.881 1.031 2.057 1.4 2.225.326.147 1.036.149 1.2-.089.059-.111.02-.351-.044-.474.277.308.651.736 1.013.942.217.104.434.17.677.18l.31-.016c.154-.033.336-.058.44-.195.116-.2.007-.756-.476-.796-.483-.04-.795-.222-1.24-.882-.365-.638.077-1.517.226-2.145.765.123 1.535.22 2.31.222.336-.017.67-.03 1.001-.093.106.27.402 1.025.404 1.239.007.601-.219 1.205-.121 1.807.06.177.005.512.35.526l.388.018.267-.008c.341.573.637.572 1.307.591zm-7.518-1.66-.063-.056c-.184-.198-.66-.544-.572-.865.075-.238.213-.457.323-.683l-.004.023c-.02.282-.059.56.032.837.278.228.663.59.918.837-.138-.038-.4-.117-.53-.066l-.104-.026z" clip-rule="evenodd"/></svg>
```

Comment (PratyushJayachandran) [reply icons/tiddlywiki.svg:1]: The improper scaling should be fixed now

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
Please merge the suggested changes
platformio

Please review the attached preview @Jermolene

File: icons/tiddlywiki.svg (hunk 1 above)
Comment (oeyoews) [reply icons/tiddlywiki.svg]: Thanks!

**Comment by oeyoews** [comment]
image

**Comment by Jermolene** [comment]
Thanks @oeyoews @PratyushJayachandran

File: _data/simple-icons.json [hunk 2]
```diff
@@ -12016,6 +12016,11 @@
             "hex": "000000",
             "source": "https://tidal.com"
         },
+        {
+            "title": "Tiddlywiki",
+            "hex": "111111",
+            "source": "https://talk.tiddlywiki.org/uploads/default/original/2X/d/ded4d2d4816bd6663a4ac467936d97f08af5fef8.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:12022]:
```suggestion
            "source": "https://tiddlywiki.com"
```

The SVG of the logo is available on the 'loading' screen of the main site.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon itself is a perfect match, thanks @oeyoews and @PratyushJayachandran for your work on this! One suggestion regarding the source, and we can get this merged in!

File: _data/simple-icons.json [hunk 3]
```diff
@@ -12112,6 +12112,11 @@
             "hex": "000000",
             "source": "https://tidal.com"
         },
+        {
+            "title": "Tiddlywiki",
```
Comment (adamrusted) [inline _data/simple-icons.json:12116]:
```suggestion
            "title": "TiddlyWiki",
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
One final note, and it's just the capitalisation of the title. Other than that, we're good to go!

**Review by adamrusted — APPROVED** [review]

**Comment by Jermolene** [comment]
Thank you @adamrusted @PratyushJayachandran @oeyoews

**Pull request merged**